engine/standalone/engine/src/main/scala/pl/touk/nussknacker/engine/standalone/management/StandaloneProcessManager.scala

```diff
@@ -200,6 +201,8 @@ class StandaloneProcessManagerProvider extends ProcessManagerProvider {
   override def createProcessManager(modelData: ModelData, config: Config): ProcessManager
     = StandaloneProcessManager(modelData, config)
 
+  override def createQueryableClient(config: Config): Option[QueryableClient] = None
```

Comment:
maybe we should also drop unused flags below? supportsSignals/QueryableState? Or let's use them